ci/deps/azure-36-locale.yaml

```diff
@@ -27,6 +27,9 @@ dependencies:
   - xlrd=1.1.0
   - xlsxwriter=0.9.8
   - xlwt=1.2.0
+  # lowest supported version of pyarrow (putting it here instead of in
+  # azure-36-minimum_versions because it needs numpy >= 1.14)
+  - pyarrow=0.12
```

Comment:
-1 on this, it's already too difficult to know what we are testing, having things like this makes it even trickier.

Since I think we are happy to require numpy 1.14 or 1.15, I'd first implement #29512 and then have this in the minimum versions build

Reply:
Can we just put that as a todo item in the "bump numpy" issue then? We currently have almost no coverage of pyarrow testing, I would prefer not waiting on the other issue for that.

Reply:
i agree, let's get coverage; bumping numpy is non-trivial and could take a bit of work